Tests/KituraWebSocketTests/ConnectionCleanupTests.swift

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import XCTest
+import Foundation
+
+import LoggerAPI
+@testable import KituraWebSocket
+import Socket
+
+class ConnectionCleanupTests: KituraTest {
+    
+    static var allTests: [(String, (ConnectionCleanupTests) -> () throws -> Void)] {
+        return [
+            ("testSingleConnectionTimeOut", testSingleConnectionTimeOut),
```

Comment:
Can you make the order of the elements in the array match the order of the functions in the file please.